packages/iris-grid/src/IrisGridRenderer.js

```diff
@@ -81,6 +81,25 @@ class IrisGridRenderer extends GridRenderer {
     this.drawPendingRowLine(context, state);
   }
 
+  drawCellContent(context, state, column, row) {
+    const { metrics, model } = state;
+    const { modelColumns, modelRows } = metrics;
+    const modelRow = modelRows.get(row);
+    const modelColumn = modelColumns.get(column);
+    const value = model.valueForCell(modelColumn, modelRow);
+    if (
+      value == null &&
```

Comment:
I think just change this to `===` instead of `==` and that will get rid of the flashing.